<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MW</class>
 <widget class="QMainWindow" name="MW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>744</width>
      <height>439</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>İş Emri Takibi</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeView" name="projectsView">
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addCompany">
          <property name="text">
           <string>Araç Konumu</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addProject">
          <property name="text">
           <string>İş Emri Ekle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addMember">
          <property name="text">
           <string>Araç Ekle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editProject">
          <property name="text">
           <string>Düzenle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeProject">
          <property name="text">
           <string>Sil</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Araçlar</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Ara</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="searchEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeView" name="personsView">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="addPerson">
          <property name="text">
           <string>Araç Ekle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editPerson">
          <property name="text">
           <string>Düzenle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removePerson">
          <property name="text">
           <string>Sil</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
